assets/tests/hooks/useShapes.test.ts

```diff
@@ -34,15 +34,25 @@ describe("useRouteShapes", () => {
     expect(result.current).toEqual([])
   })
 
-  test("loads a subway route shape from hardcoded data", () => {
-    const mockFetchShape: jest.Mock = Api.fetchShapeForRoute as jest.Mock
+  test("loads a subway route shape from API", async () => {
+    const shape = shapeFactory.build({ className: undefined })
+    const mockFetchShape = jest
+      .mocked(Api.fetchShapeForRoute)
+      .mockReturnValueOnce(instantPromise([shape]))
```

Comment:
suggestion(non-blocking): what do you think about using `mockResolvedValueOnce` instead?

```suggestion
      .mockResolvedValueOnce([shape])
```